airflow-core/src/airflow/ui/src/components/PoolBar.tsx

```diff
@@ -42,13 +42,28 @@ export const PoolBar = ({
         const slotValue = pool[key];
         const flexValue = slotValue / totalSlots || 0;
 
+        // Hide empty segments
         if (flexValue === 0) {
           return undefined;
         }
 
+        // NEW: If this segment represents "deferred_slots" and the current pool
```

Comment:
Don't put things like "NEW" or "changed" etc in code comments. As soon as the PR is merged the comment is instantly out of date.